busybox/job.json

```diff
@@ -15,12 +15,23 @@
           }
         ]
       },
-      "taskCount": 3,
-      "taskEnvironments": {
-        "variables": {
-          "BUCKET": "batch-test-bucket"
+      "taskEnvironments": [
+        {
+          "variables": {
```

Comment:
I think we should add the environment section as part of the runnable definition if we're passing the same value. In that case we can keep `taskCount` It'd look like this:

```json
        "runnables": [
          {
            "container": {
              "imageUri": "busybox",
...
            },
            "environment": {
              "variables": { ... }
            }
          }
```

What do you think?

Reply:
Good catch @yuryu! I thought `environment` was deprecated but this was for another field. I now changed both samples to use `environment` instead. Definitely makes the samples simpler.